aiohttp/connector.py

```diff
@@ -467,30 +481,57 @@ def family(self):
         return self._family
 
     @property
+    def cache_dns(self):
```

Comment:
Worth to. Btw, on second thought I found myself incorrect about the name: seems like this property is used as True/False flag in order to define will dns cache be used or not, right? Then `use_dns_cache` should be semantically better and won't cause confusion about if it some sort of cache mapping or not. What do you think?